appcenter-analytics/Analytics.d.ts

```diff
@@ -0,0 +1,6 @@
+export function trackEvent(
+  eventName: string,
+  properties?: { [name: string]: string | number | boolean }
+): Promise<string | null>;
+export function isEnabled(): Promise<boolean | string>;
```

Comment:
The promise is boolean only, not string

Reply:
I read https://docs.microsoft.com/en-us/appcenter/sdk/analytics/react-native#check-if-app-center-analytics-is-enabled as that it'd return a string (error) if it fails. Does it throw an error if it fails?

Reply:
This API cannot fail, its asynchronous but just a getter for a boolean value.

Reply:
👍 fixed
Might be worth adjusting the docs for this too, made a pull request there (https://github.com/MicrosoftDocs/appcenter-docs/pull/37).